packages/tailwindcss/src/index.ts

```diff
@@ -45,19 +56,24 @@ function parseThemeOptions(selector: string) {
 
 async function parseCss(
   css: string,
-  { loadPlugin = throwOnPlugin, loadConfig = throwOnConfig }: CompileOptions = {},
+  base: string,
+  { loadModule = throwOnLoadModule, loadStylesheet = throwOnLoadStylesheet }: CompileOptions = {},
 ) {
-  let ast = CSS.parse(css)
+  let ast = [context({ base }, CSS.parse(css))] as AstNode[]
+
+  if (css.includes('@import')) {
```

Comment:
Shower thought but I'd kinda expect almost every CSS file to include an `@import 'tailwindcss'` to some degree. So maybe we should skip this check and rather pay a cost for a file that does not import tailwindcss? 🤔

Reply:
Definitely remove the check. The defacto way to import tailwind v4 is `@import "tailwindcss";` and if we don't flatten before compile sees it the code is always going to have to run anyway.